src/main/java/core/basesyntax/FileWork.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.File;
+import java.io.FileReader;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
 public class FileWork {
-    public String[] readFromFile(String fileName) {
-        return null;
+    private static final String START_LETTER = "w";
+    private static final String[] SIGNS = new String[] {
+            ",","!","?","."
+    };
+
+    public String[] readFromFile(String firstFileName) {
+        File file = new File(firstFileName);
+        List<String> arrayOfWords = new ArrayList<>();
+
+        try {
+            BufferedReader bufferedReader = new BufferedReader(new FileReader(file));
+            String value = bufferedReader.readLine();
+            while (value != null) {
+                for (String word : value.split(" ")) {
```

Comment:
```suggestion
                for (String word : value.split("\\W+")) {
```
this way you don't need to cut your word with substring and variable SIGNS will not be needed